### PR TITLE
Added new Gradle task for games ran from client.

### PR DESCRIPTION
**Pull request by jbloxham**
Resolves https://github.com/battlecode/battlecode-client-17/issues/113 . See https://github.com/battlecode/battlecode-server/pull/377 .

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,59 +10,62 @@ targetCompatibility = 1.8
...
 if (awsCredentials != null) {
-  publishing {
-    repositories {
-      maven {
-        name 's3'
-        url 's3://battlecode-maven'
-        credentials(AwsCredentials) {
-          accessKey awsCredentials.AWSAccessKeyId
-          secretKey awsCredentials.AWSSecretKey
+    publishing {
```
Comment (kazimuth) [inline build.gradle:58]: Can we publish to both?

Comment (jbloxham) [reply build.gradle:58]: tru

Comment (jbloxham) [reply build.gradle:58]: actually apparently this is surprisingly annoying: http://stackoverflow.com/questions/17187577/gradle-upload-a-single-archive-to-multiple-repositories

Comment (kazimuth) [reply build.gradle:58]: It'd still be best to have it, the example code doesn't look too terrible.

Comment (kazimuth) [reply build.gradle:58]: Resolved, it turns out 'publish' is different from 'upload'

**Review by kazimuth — APPROVED** [review]

**Pull request merged**